### ISSUE TITLE
TaskListService with Detailed=true not returning StartTime and RunningTime

### ISSUE THREAD

**Issue by jesstracy** [olivere/elastic#769]
### Which version of Elastic are you using?
[ ] elastic.v6 (for Elasticsearch 6.x)

### Please describe the expected behavior
When I call the TaskListService with Detailed field set to true, I think I should be able to access StartTime and RunningTime of the returned tasks

### Please describe the actual behavior
I get the Id and and Action fields of the returned tasks, but StartTime and RunningTime are empty strings

### Any steps to reproduce the behavior?
```
taskListResp, err := elastic.NewTasksListService(client).
        Detailed(true).
        Do(ctx)
if err != nil {
    fmt.Println(err)
}

for _, node := range taskListResp.Nodes {
    for _, task := range node.Tasks {
        fmt.Println("Id:", task.Id)
        fmt.Println("Action:", task.Action)
        fmt.Println("StartTime:", task.StartTime)     //why this empty
        fmt.Println("RunningTime:", task.RunningTime) //why this empty
    }
}
```

Thanks!

**Comment by jesstracy** [comment]
Update: Still not sure why those two fields are blank, but I am able to access RunningTimeInNanos, which is info enough to do what I need to do. Thanks.

**Comment by olivere** [comment]
@jesstracy Hi! Setting `detailed=true` only gives you some more fields. Using `human=true` will give you the human-readable fields in nearly all Elasticsearch APIs with time/size fields. However, the `Human(...)` setter is missing in the Task List API, so I re-opened this and will fix it in the next release.

```
curl 'http://localhost:9200/_tasks?detailed=true&pretty=true&human=true'
````

**Comment by olivere** [comment]
Will be fixed in 6.1.19 by e7a6ef5484f705136641b74a817d7ea73b4128c4.
